.github/workflows/rust.yml

```diff
@@ -22,9 +22,11 @@ concurrency:
   cancel-in-progress: true
 
 on:
-  # always trigger on PR
   push:
   pull_request:
+    paths-ignore:
+      - "docs/**"
```

Comment:
🤔  I think that the `prettier` checks run for doc changes

https://github.com/apache/arrow-datafusion/blob/master/.github/workflows/dev.yml#L39-L57

Perhaps we can add the trigger condition for that job so the markdown is checked when md files are modified?

Reply:
I think `prettier` checks will run for doc changes, here we only don't trigger rust ci for doc changes.

Here is a test: https://github.com/xudong963/arrow-datafusion/pull/3